src/include/common/expression_type.h

```diff
@@ -24,6 +24,7 @@ const std::string CAST_TO_DATE_FUNC_NAME = "DATE";
 const std::string CAST_TO_TIMESTAMP_FUNC_NAME = "TIMESTAMP";
 const std::string CAST_TO_INTERVAL_FUNC_NAME = "INTERVAL";
 const std::string CAST_TO_STRING_FUNC_NAME = "STRING";
+const std::string CAST_INT64_TO_DOUBLE_FUNC_NAME = "INT64_TO_DOUBLE";
```

Comment:
I think we only need a CAST_TO_DOUBLE function and overload it with different input types

Reply:
renamed